camelComponent/README.md

```diff
@@ -94,6 +94,54 @@ By adding the `structuredMessageHeader` component endpoint option with a value o
 
 A schema type definition for this message structure is available in `src/main/resources/types/com/vantiq/extsrc/camelcomp/message.json`.
 
+#### Header Duplication
+
+There are numerous components in the Apache Camel universe, and many include Camel Headers that convey the same 
+information. For example, a number of components include headers whose name is of the form `Camel<componentn 
+name>ContentType` (_e.g._, `CamelAzureServiceBusContentType` to name but one). 
+
+If the Vantiq application wants to deal with these, it does not want to know all possible variations here -- that is,
+after all, the purpose of an integration system: to manage these discrepancies.  To ease the handling of these 
+situation, the Vantiq Camel Component can duplicate (NOT replace) these header values into alternative names. When 
+this is requested, the original set of headers will be sent or received, but the duplicates with values will be sent 
+or received as well.
+
+To do this, the Camel system must be configured with a Bean of type `io.vantiq.extsrc.camel.HeaderDuplicationBean`, 
+containing a Map<String, String> where the keys of the map represent the header names to be duplicated, and the 
+values are the new header names into which the header value will be duplicated. (Note: we do this via a configured 
+bean since the number of headers could be large, and having URI of great length is sometimes problemnatic.)
+
+To use this, add the `headerDuplicationBeanName` component/endpoint option with the value being the name of the bean 
+containing the map described above. Note that this header duplication is performed only when the 
+`structuredMessageHeader` is present with the value `true`. In other cases, it is silently ignored.
```

Comment:
missing word?   property or setting as in:   _the structuredMessageHeader property_
```suggestion
`structuredMessageHeader` property is present with the value `true`. In other cases, it is silently ignored.
```